README.md

```diff
@@ -1,13 +1,35 @@
-**You are NOT supposed to push to this repo.** 
+# The Klaytn Docs
 
+[![Gitter](https://badges.gitter.im/klaytn/klaytn-docs.svg)](https://gitter.im/klaytn/klaytn-docs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-The data flow is as below. 
+![Klaytn Docs](images/klaytn.png)
 
-`klaytn-docs:master` -> `Crowdin` -> `klaytn-docs:l10n`
+Klaytn Docs introduces [Klaytn](https://klaytn.foundation/), the blockchain of choice for web3 era with high performance and great user experience. In these documents you will find detailed explanations on Klaytn along with guidelines and instructions on how to use Klaytn and interact with the Klaytn network. More information on how to use Klaytn Docs is depicted on the [Klaytn Docs Wiki](https://github.com/klaytn/klaytn-docs/wiki)
 
-- If you want to change the source content, you should update the `master` branch of `klaytn-docs`.
-- If you want to change the translation, you should work on [Crowdin](https://crowdin.com/project/klaytn-docs). Then the modified translation will be pushed back to the `l10n` branch of `klaytn-docs`. To request access to the translation, follow the link - https://crwd.in/klaytn-docs. 
-- Do NOT push to `klaytn-docs:l10n` directly. It will be overwritten by Crowdin and the change will get lost.
+## Want to Contribute to the Translation? <a id="want-to-translate"></a>
 
-`dev` branch of this repo mirrors the `ko/docs` subfolder of the `klaytn-docs:l10n` periodically. Approved contents will be merged to `master` branch and published at https://ko.docs.klaytn.com. 
+Thank you for your interest in Klaytn Docs! As an open source project, Klaytn Docs is open for contribution by anyone, which means you can also participate in the translation of the English version into other languages.
 
+However, **you are not supposed to push to this repo directly.** 
+
+Instead, you can contribute to the translation on the [Crowdin](https://crowdin.com/project/klaytn-docs) platform.
+
+**Workflow summary**
+
+- Submitted translations get approved by our maintainers.
+- The approved translations will be pushed to the `l10n` branch of `klaytn-docs`. The `dev` branch of this repo mirrors the `ko/docs` subfolder of `klaytn-docs:l10n` periodically. 
+- Our maintainers will merge the changes to the `master` branch, which will be reflected at https://ko.docs.klaytn.com. 
+
+Note: **Do NOT push to `klaytn-docs:l10n` directly.** It will be overwritten by Crowdin and the changes will get lost.
+
+To start translating, follow the link - https://crwd.in/klaytn-docs. 
```

Comment:
Seems that link is broken?

Reply:
@kjhman21  I fixed the link! Can you approve now? 😃